tests/ext/catboost/test_model.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import pytest
+from catboost import CatBoostClassifier
+
+from ebonite.core.analyzer.model import ModelAnalyzer
+
+
+@pytest.fixture
+def catboost_model(pandas_data):
+    return CatBoostClassifier(iterations=1).fit(pandas_data, [1, 0])
+
+
+def test_catboost_model_wrapper(catboost_model, pandas_data, tmpdir):
+    # this import is required to ensure that CatBoost model wrapper is registered
+    import ebonite.ext.catboost  # noqa
+
+    cbmw = ModelAnalyzer.analyze(catboost_model)
+
+    assert cbmw.model is catboost_model
+
+    with cbmw.dump() as artifact:
+        artifact.materialize(tmpdir)
+    cbmw.load(tmpdir)
```

Comment:
you can add .unbind_model and test that .predict will raise ValueError to be sure that .load is actually working

Reply:
done